Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson4
+    {
+        public static void Demo()
+        {
+            //run twice
```

Comment:
Redundant comment- not needed.

Reply:
Write code that speaks for itself, do not use comments for explaining blocks of code.
Use step functions to explain blocks of code.